app/models/exporters/csv_exporter.rb

```diff
@@ -44,7 +44,12 @@ def headers(fields)
 
   def row(record, fields)
     [record.id] + fields.map do |field|
-      record.data[field.name]
+      next record.data[field.name] unless field.form.unique_id.include?('_subform_section')
+
+      subform_field_name = field.form.unique_id.gsub('_subform', '')
```

Comment:
Subform field names are not necessarily the same as the subform section unique_id.

Reply:
How can I get values for sub-form fields here?